### ISSUE TITLE
Fear as a stat?

### ISSUE THREAD

**Issue by CrankyUnicorn** [CleverRaven/Cataclysm-DDA#40574]
I fear a lot when I play CDDA, the fear of loosing the player character is what pumps my adrenaline and makes playing the game fun. But it is also logic that the character would feel fear too and loose focus when seeing strange and bizzare that he or she never deal with or killed or corpses or walking in the pitch dark of a lab? Personally I would poop my pants and die just seeing something like a Mi-go.

**Comment by CrankyUnicorn** [comment]
By Aluminumfoil
"I was going to say that it should have a max duration or something, but being terrified, unable to sleep until dead tired, unable to concentrate, for days because of witnessing too much horror all at once actually sounds... just about right."

**Comment by Aphegis** [comment]
In my opinion this would be really boring, if anyone implements this in the future please make it optional, not to mention that it would require a LOT of work to make it good.

**Comment by Barhandar** [comment]
There's already morale loss; CDDA is a full direct control game (i.e. the character will only do things you command them to and will _not_ do things you did not command) and thus, IMO, without changing this paradigm "fear" would just be "morale but slightly different" and hence superfluous.

**Comment by CrankyUnicorn** [comment]
To Aphegis: Yes it may result in a boring mechanic but we only know when we try it. I concord with you that should be optional I even say it should be directly related to a Negative Trait like Scary Cat , Fearful or Fear Paralysis.   

To Barhandar: I discord of that point since you are not always in control of the player. If the moral is too low the player will not butcher, if the player is in pain it will not move as fast as you would like, if you trying to sleep and he ears something he is not going to sleep. Yes the Moral system does already some of the work and funny is the same name that they gave to it in the original X-COM (Moral) but IRL they are different things. 

I see Fear relate to the Cognition and Consciousness as Pain is to the Body and Limbic System.
So let me rephrase instead or relate it directly to Moral I proposes to relate it parallel to PAIN since it more logic to be a cause rather then a consequence. So the value of Pain and Fear should then be calculated into Moral and Movement.
